Firestore/core/src/firebase/firestore/util/CMakeLists.txt

```diff
@@ -16,69 +16,66 @@
 # libraries in here are an implementation detail of making this a
 # mutli-platform build.
 
-add_library(
+cc_library(
   firebase_firestore_util_base
-  secure_random_arc4random.cc
-  string_printf.cc
-)
-target_link_libraries(
-  firebase_firestore_util_base
-  PUBLIC
-  absl_base
+  SOURCES
+    secure_random.h
+    secure_random_arc4random.cc
+    string_printf.cc
+    string_printf.h
+  DEPENDS
+    absl_base
 )
 
+## assert and log
+
 # stdio-dependent bits can be built and tested everywhere
-add_library(
-  firebase_firestore_util_stdio
-  assert_stdio.cc
-  log_stdio.cc
-)
-target_link_libraries(
+cc_library(
   firebase_firestore_util_stdio
-  PUBLIC
-  firebase_firestore_util_base
+  SOURCES
+    assert_stdio.cc
+    log_stdio.cc
+  DEPENDS
+    firebase_firestore_util_base
+    absl_base
 )
 
 # apple-dependent bits can only built and tested on apple plaforms
```

Comment:
You've dropped 'if(APPLE)'. Was that on purpose? If so, I'd recommend updating this comment.

Update: Ah, I see you have an ifAPPLE down on line 61 that effectively does this. It's probably simplest to just move this comment down there. Or alternatively, just delete the comment; the 'ifAPPLE' seems descriptive enough.

Reply:
I've deleted the comment though I'm still not sure this is the best approach. I was hoping that there would at least be some clue as to why we're excluding these targets from 'all'.